dockstore-common/src/test/java/io/dockstore/common/yaml/DockstoreYamlTest.java

```diff
@@ -198,4 +202,81 @@ public void testGalaxySubclass() throws DockstoreYamlHelper.DockstoreYamlExcepti
         assertEquals(4, workflows.stream().filter(w -> w.getSubclass().equalsIgnoreCase("gxformat2")).count());
     }
 
+    @Test
+    public void testGitReferenceFilter() {
+        assertTrue(DockstoreYamlHelper.filterGitReference(Path.of("anything"), List.of()));
+
+        // Generic name filter
+        // Glob
+        assertTrue(DockstoreYamlHelper.filterGitReference(Path.of("refs/heads/develop"), List.of("develop")));
+        assertTrue(DockstoreYamlHelper.filterGitReference(Path.of("refs/tags/develop"), List.of("develop")));
+        assertTrue(!DockstoreYamlHelper.filterGitReference(Path.of("refs/heads/foo/develop"), List.of("develop")));
```

Comment:
I think these should be assertFalse -- I was going to write a comment about how you should have some negative tests.